models/blog-post.ini

```diff
@@ -33,4 +33,4 @@ width = 1/4
 [fields.body]
 label = Body
 description = reStructuredText
```

Comment:
I'm guessing these description fields can be updated?

Reply:
(they seem to be in all the models, plus the flow block definition)

Reply:
Yes! I meant to do that and missed it.